### ISSUE TITLE
1.12 ?

### ISSUE THREAD

**Issue by JK82NL** [MineLittlePony/ItemDash#3]
This might not be the place to ask, but since I cannot find any forum for this mod. Are you planning on updating this? If I am correct this is already implemented in the minelittlepony mod, isnt it?
I've been waiting for this one since I like it much better then Toomanyitems and all the others.

**Comment by mattmess1221** [comment]
I suggest to just use JEI. But if enough people want it, I'll update it.

**Comment by saturnux** [comment]
i dont like jei, but if you update itemdash, i use it.

**Comment by Phoenix616** [comment]
JEI needs Forge. That's the cool thing about ItemDash, it only needs LiteLoader! :D

**Comment by mattmess1221** [comment]
What do you mean "only needs LiteLoader"? JEI also "only needs Forge." What's the point you're making?

**Comment by Phoenix616** [comment]
The point I'm making is that it is the only item-mod (that I know of) that is a LiteMod. The reason i said "only" is because LiteLoader is, as the name says, a lot liter than Forge.

My daily setup is completely build on LiteLoader mods as using Forge might cause some incompatibilities and all the features I need can be found in LiteMods.

**Comment by mattmess1221** [comment]
Yeah, I'll get to it at some point then.

**Comment by JK82NL** [comment]
Yes, I agree with Phoenix616. I also think alot of people don't even know about this mod since they can only find it through the Liteloader page. If there was a minecraftforum page for instance  it would get alot more attention.

You said "I suggest to just use JEI. But if enough people want it, I'll update it."  but that is kinda hard since it is only found here on github, isn't it?

*edit* We were probably writing our posts at the same time so I missed yours. Thank you, I am happy to hear you will continue this mod.

**Comment by HowardZHY** [comment]
would anyone upgrade this to 1.12.2 or make a better 1.8.9 version? TMI was dead and NEI sucks

**Comment by mattmess1221** [comment]
@HowardZHY What features are you missing from NEI or JEI?

**Comment by HowardZHY** [comment]
> @HowardZHY What features are you missing from NEI or JEI?

there's no NEI for 1.8.9 , and I just want grab items quickly and without check recipe
